src/components/common/modal/ModalJaggedEdge.js

```diff
@@ -1,19 +1,23 @@
 // @flow
 import React from 'react'
-import { View } from 'react-native'
+import { Platform, View } from 'react-native'
 import { withStyles } from '../../../lib/styles'
 import { mediumZIndex } from './styles'
 
 const ModalJaggedEdge = ({ styles, style }: any) => <View style={[styles.jaggedEdge, style]} />
 
 const getStylesFromProps = ({ theme }) => ({
   jaggedEdge: {
-    backgroundImage: `linear-gradient(40deg, transparent 75%, ${
-      theme.modals.backgroundColor
-    } 76%), linear-gradient(-40deg, transparent 75%, ${theme.modals.backgroundColor} 76%)`,
-    backgroundPosition: `-${theme.modals.jaggedEdgeSize / 2}px 0`,
-    backgroundRepeat: 'repeat-x',
-    backgroundSize: `${theme.modals.jaggedEdgeSize}px ${theme.modals.jaggedEdgeSize}px`,
+    ...Platform.select({
+      web: {
```

Comment:
what do we use then for native?

Reply:
native needs different styling, a few months ago we already discussed this, Mathias tried to get the same look using react-native-linear-gradient but couldn't, we agreed on leaving it for the future with either a static image repeated or a different lib.

Reply:
then open an issue for that. both here and in the other shadowbox comment you simply ignore it is there and remove it, like no further action is required.